### PR TITLE
ROX-13887: Revert rules to previously applied YAML

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR is an iterative step to adding the network policy simulator to the network graph. This PR includes:
1. Updating the `NetworkPolicySimulatorSidePanel` to show the previously applied YAML when we are in the `UNDO` state of the network policy simulator.

Screenshot 2022-12-16 at 7 21 11 AM
Screenshot 2022-12-16 at 7 21 18 AM

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### CODE REVIEW

File: ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx [hunk 1]
```diff
@@ -53,17 +63,15 @@ function NetworkPoliciesYAML({ yaml, generateNetworkPolicies }: NetworkPoliciesY
...
-                    </Title>
-                </EmptyState>
-            </Bullseye>
-        );
-    }
+    const revertRecentYAML = undoNetworkPolicies ? (
+        <CodeEditorControl
+            icon={<UndoIcon />}
+            aria-label="Revert most recently applied YAML"
+            toolTipText="Revert most recently applied YAML"
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx:70]: nit: If these are the same, perhaps we should define a string constant and reuse it in both props.

File: ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx [hunk 2]
```diff
@@ -53,17 +63,15 @@ function NetworkPoliciesYAML({ yaml, generateNetworkPolicies }: NetworkPoliciesY
...
-            <Bullseye>
-                <EmptyState variant={EmptyStateVariant.xs}>
-                    <Title headingLevel="h4" size="md">
-                        No network policies
-                    </Title>
-                </EmptyState>
-            </Bullseye>
-        );
-    }
+    const revertRecentYAML = undoNetworkPolicies ? (
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx:66]: Why is this optional?

**Review by vjwilson — APPROVED** [review]
Just a couple of minor questions.

File: ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx (hunk 2 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx:66]: Hmm, actually that's a good question. I think the controls should be visible regardless and not hidden when viewing active YAMLs. I'll change that

**Pull request merged**